learning_assistant/views.py

```diff
@@ -76,6 +78,21 @@ def post(self, request, course_run_id):
         unit_id = request.query_params.get('unit_id')
 
         message_list = request.data
+
+        # Check that the last message in the list corresponds to a user
+        new_user_message = message_list[-1]
+        if new_user_message['role'] != LearningAssistantMessage.USER_ROLE:
+            return Response(
+                status=http_status.HTTP_400_BAD_REQUEST,
+                data={'detail': "Expects user role on last message."}
+            )
+
+        course_id = get_course_id(course_run_id)
+        user_id = request.user.id
+
+        if chat_history_enabled(course_id):
```

Comment:
Should this be the course_id or a course key?

Reply:
Good catch. According to the code to check the flag we [check for Learning Assistant enabled state](https://github.com/edx/learning-assistant/blob/e28918a6a682719c72533f0435dcd76488ff0e1b/learning_assistant/views.py#L158) is a CourseKey based on the course run. I'll update it quickly.